Debian-specific tweak: the pointerMode resource is set to 0 (never)
by default.

Index: xterm/xterm.man
===================================================================
--- xterm.orig/xterm.man
+++ xterm/xterm.man
@@ -2663,11 +2663,11 @@
 .RS
 .TP 3
 0
-never
+never.
+This is the default.
 .TP 3
 1
 the application running in \fIxterm\fP has not activated mouse mode.
-This is the default.
 .TP 3
 2
 always.
